dataseeding/sandbox.js

```diff
@@ -0,0 +1,6 @@
+const faker = require('faker');
+
+// console.log(faker.hacker.phrase());
+
+console.log;
```

Comment:
Is this sandbox.js file necessary? In your project's current state, I don't understand this file's purpose.